apps/macOS/FluentUIReactNativeTester_macOS/Config/FluentUIReactNative_common.xcconfig

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+PRODUCT_MODULE_NAME = FluentUIReactNativeTester
+SWIFT_MODULE_NAME = FluentUIReactNativeTester
+
+ALWAYS_SEARCH_USER_PATHS = NO
+CLANG_ENABLE_MODULES = YES
+
+// Versions
+MACOSX_DEPLOYMENT_TARGET = 10.14
```

Comment:
Maybe can handle in a follow when we figure out how to deal with ios: I think this is the only platform-specific config, right? Whereas all the other stuff applies to both macOS and iOS and thus should be shared with iOS?

Again, probably best as a "file away and remember" instead of architecting too strictly now before we know what the iOS/macOS split looks like unless you can think of something both cheap and flexible

Reply:
I'll think about it but agree it shouldn't stop this from going in